src/main/java/org/fogbowcloud/manager/core/constants/ConfigurationConstants.java

```diff
@@ -2,11 +2,21 @@
 
 public class ConfigurationConstants {
 
+    // CONFIGURATION FILES PATHS
+	public static final String MANAGER_CONF_FILE_FULL_PATH = "src/main/resources/manager.properties";
+
+	public static final String INFRA_CONF_FILE_FULL_PATH = "src/main/resources/infrastructure.properties";
+
+	public static final String FEDERATION_CONF_FILE_FULL_PATH = "src/main/resources/federation.properties";
+
+    // XMPP
 	public static final String XMPP_ID_KEY = "xmpp_jid";
 
+	public static final String XMPP_HOST_KEY = "xmpp_host";
```

Comment:
I do not undestand the meaning of this configuration constant.

Reply:
Is it the XMPP server IP address. I was using in an old test, but no need using anymore.